index.html

```diff
@@ -16,7 +16,7 @@
             <img src="images/brand.png">
           </a>
         </div>
-        <form class="navbar-form">
+        <form action="javascript:void(0);" class="navbar-form">
```

Comment:
please make this form tag into a div (or some other tag) so that you don't have to define an action attribute on the tag.